docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_grafana/_index.md

```diff
@@ -0,0 +1,170 @@
+
+- You will have the Azure Monitor Agent (AMA) deployed on your Windows or Linux system and reporting to the Log Analytics workspace that has been created. You can verify by going back to your Azure Arc-enabled server, **Extensions** section:
+
+    ![Screenshot AMA extension on Windows](./07.png)
+
+- Moreover, a **Data Collection Rule (DCR)** is created to send logs from the Azure Arc-enabled servers to the new **Log Analytics workspace**. In addition the **Azure Managed Grafana** service is deployed which we will explore later in this scenario.
+
+    ![Screenshot Data Collection Rules and Log Analytics workspace](./08.png)
+
+- Note that the template deployment also created a required role assignment for **Azure Managed Grafana** to be able to access your monitoring data from its **Managed Identity**:
+
+    ![Screenshot Data Collection Rules and Log Analytics workspace](./08_2.png)
+
+- If you click on the **Data Collection Rules (DCR)**, you will see the **Resources** attached to it and the collected **Data Sources**.
+
+  - On the **Resources** blade, you will see your Windows Azure Arc-enabled server:
+
+    ![Screenshot Windows Data Collection Rules - Resources](./09.png)
+
+  - On the **Data Sources** blade, you will see *Performance Counters*
+
+    ![Screenshot Windows Data Collection Rules - Perf Counters](./10.png)
+
+  - If you click on *Performance Counters*, you will see the **Data source** that is collected and the **Destination**, which is the Log Analytics workspace created as part of this scenario:
+
+    ![Screenshot Windows Data Collection Rules - Perf Counters - Data Source](./11.png)
+
+    ![Screenshot Windows Data Collection Rules - Perf Counters - Destination](./12.png)
+
+- Go back to your **resource group** and click on the **Log Analytics Workspace**:
+
+  ![Screenshot Log Analytics workspace](./13.png)
+
+- Click on **Logs**:
+
+  ![Screenshot Log Analytics workspace - Logs](./14.png)
+
+- **Run** the following **query**. It will show you the **data types collected** by the **Azure Monitor Agent (AMA)** on each machine by using the **Data Collection Rules (DCR)**:
+
+  ```shell
+  search * 
+  | distinct Computer, Type
+  | where Type != "Heartbeat" and Type != "Usage"
+  | sort by Computer asc
+  ```
+
+  ![Screenshot Log Analytics workspace - Query](./15.png)
+
+- It is time to explore **Azure Managed Grafana** and the included sample dashboard. Go back to your **resource group** and click on **Azure Managed Grafana**:
+
+  ![Screenshot Azure Managed Grafana resource group](./16.png)
+
+- In the **Azure Managed Grafana** overview click on the **Endpoint** url to open the Grafana webinterface.
+
+> **NOTE: Authenticate to Grafana with the user you provided in the template deployment (grafanaAdminprincipalId) or add your current user to one of the Azure RBAC roles Grafana Admin, Grafana Editor, Grafana Viewer**.
+
+  ![Screenshot Azure Managed Grafana endpoint](./17.png)
```

Comment:
for accessibility purposes, provide a descriptive image title. 17.png, 18.png, 19.png and 20.png have the same title. Same happens for 03.png and 04.png